docs/user-guide/config-cert-instance-hints.md

```diff
@@ -0,0 +1,54 @@
+Hints about creating the certificate and instance when configuring Zowe.
+
+When you are configuring Zowe on z/OS, you need to create certificates, and then create the Zowe instance.
+
+### Creating certificates
+
+We recommend that you re-create the certificates after every Zowe upgrade.
```

Comment:
We don't currently recommend this - keystore generation is listed as a one-time task

Reply:
Ok, then I misunderstood.  Is there one keystore per instance then?

Reply:
The new HOLDDATA says this - is this a conflict?
```
Run <ROOT_DIR>/bin/zowe-setup-certificates.sh again with an 
updated zowe-setup-certificate.env to pick up the new 
definitions.
```

Reply:
1.12 has some extra parameters - hence is in the HOLDDATA, but it isn't mandatory, or usual

Reply:
> Ok, then I misunderstood. Is there one keystore per instance then?

No

Reply:
So, 
1.  normally, you run `zowe-setup-certificates.sh` only the first time you install Zowe, unless instructed otherwise by HOLDDATA or the release notes for that release.
2.  the keystore is shared by all instances.

Reply:
1 - yes
2 - it doesn't have to be, but that is the way Joe has doc'd it and our default recommendation if you create the keystore directory in /global/zowe/keystore